vpype/hpgl_devices.toml

```diff
@@ -40,3 +40,38 @@ y_range = [0, 10365]
 y_axis_up = true
 origin_location = ["15mm", "287mm"]
 set_ps = 0
+
+[device.calcomp_designmate]
+name = "designmate"
+plotter_unit_length = "0.02488mm"
+pen_count = 8
+
+[[device.calcomp_designmate.paper]]
+name = "a4"     # Will always be loaded in Landscape oriebtaion in the Designmate
+aka_names = ["A4"]
+paper_size = ["297mm", "210mm"]
+x_range = [0, 11040]
+y_range = [0, 7721]
+y_axis_up = true
+origin_location = ["12mm", "198mm"]
+set_ps = 0
+
+[[device.calcomp_designmate.paper]]
+name = "a3"  # As loaded in portrait orientaion in the plotter. Does not affect drawing orientaion, only page margins
+aka_names = ["A3"]
+paper_size = ["420mm", "297mm"]
```

Comment:
You must invert this definition (since this configuration is for the page loaded in portrait mode)

Reply:
The orientation of the paper when loaded does not affect the orientation of the plot nor the corner where the origin is on the page . This is what we have discussed before in the DM and is to see in the photos
![20200921_135553](https://user-images.githubusercontent.com/52834821/94716979-f7d28200-034f-11eb-9fd7-2ed238906c35.jpg)
![20200921_140419](https://user-images.githubusercontent.com/52834821/94716999-04ef7100-0350-11eb-9efd-c68e53c9189a.jpg)
This is taken care of by the auto rotation feature of the Calcomp plotter. 
It only thing affected by the loading orientation is the margin and the mm exact position of the origin. The position of the side relative to the origin remains the sames

Reply:
checked